modules/installation-creating-gcp-security.adoc

```diff
@@ -48,40 +58,97 @@ resources:
 - name: cluster-security
   type: 03_security.py
   properties:
-    infra_id: '${INFRA_ID}' <1>
-    region: '${REGION}' <2>
-
-    cluster_network: '${CLUSTER_NETWORK}' <3>
-    network_cidr: '${NETWORK_CIDR}' <4>
-    master_nat_ip: '${MASTER_NAT_IP}' <5>
-    worker_nat_ip: '${WORKER_NAT_IP}' <6>
+    allowed_external_cidr: '0.0.0.0/0' <1>
+    infra_id: '${INFRA_ID}' <2>
+    region: '${REGION}' <3>
+    cluster_network: '${CLUSTER_NETWORK}' <4>
+    network_cidr: '${NETWORK_CIDR}' <5>
+    master_nat_ip: '${MASTER_NAT_IP}' <6>
+    worker_nat_ip: '${WORKER_NAT_IP}' <7>
 EOF
 ----
-<1> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
-<2> `region` is the region to deploy the cluster into, for example `us-east1`.
-<3> `cluster_network` is the `selfLink` URL to the cluster network.
-<4> `network_cidr` is the CIDR of the VPC network, for example `10.0.0.0/16`.
-<5> `master_nat_ip` is the IP address of the master NAT, for example `34.94.100.1`.
-<6> `worker_nat_ip` is the IP address of the worker NAT, for example `34.94.200.1`.
+<1> `allowed_external_cidr` is the CIDR range that can access the cluster API and SSH to the bootstrap host. For an internal cluster, set this value to `${NETWORK_CIDR}`.
+<2> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
+<3> `region` is the region to deploy the cluster into, for example `us-east1`.
+<4> `cluster_network` is the `selfLink` URL to the cluster network.
+<5> `network_cidr` is the CIDR of the VPC network, for example `10.0.0.0/16`.
+<6> `master_nat_ip` is the IP address of the master NAT, for example `34.94.100.1`.
+<7> `worker_nat_ip` is the IP address of the worker NAT, for example `34.94.200.1`.
 
 . Create the deployment by using the `gcloud` CLI:
 +
 ----
 $ gcloud deployment-manager deployments create ${INFRA_ID}-security --config 03_security.yaml
 ----
 
+. Export the variable for the master service account:
++
+----
+$ export MASTER_SA=${INFRA_ID}-m@${PROJECT_NAME}.iam.gserviceaccount.com
```

Comment:
Look up the IAM is better than hard code the dns

    export MASTER_SA=$(gcloud iam service-accounts list --filter "email~^${INFRA_ID}-m@${PROJECT_NAME}." --format json | jq -r '.[0].email')